trace_state/HTTP_HEADER_FORMAT.md

```diff
@@ -1,14 +1,14 @@
 # Trace Context Extended HTTP Header Format
 
-A correlation context header is used to pass the name-value context properties for the trace. This is a companion header for the `Trace-Context`. The values MUST be passed along to any child requests. Note that uniqueness of the key within the `Trace-Context-Ext` is not guaranteed. Context received from upstream service may be altered before passing it along.
+A correlation context header is used to pass the name-value context properties for the trace. This is a companion header for the `Trace-Parent`. The values MUST be passed along to any child requests. Note that uniqueness of the key within the `Trace-State` is not guaranteed. Context received from upstream service may be altered before passing it along.
 
 *See [rationale document](HTTP_HEADER_FORMAT_RATIONALE.md) for details of decisions made for this format.*
 
 # Format
 
 ## Header name
 
-`Trace-Context-Ext`
+`Trace-State`
```

Comment:
`Trace-State` should make a statement about what are inside. Such as: vendor-name=vendor-context-segment . And make an example for it. Even we have not discussed about vendor-context-segment .

Reply:
Let's make it in separate PR. I want to merge renaming to minimize conflicts. File renamings makes merges nasty